packages/unmock-node/package.json

```diff
@@ -25,6 +25,7 @@
     "readable-stream": "^3.4.0",
     "shimmer": "^1.2.1",
     "unmock-core": "file:../unmock-core",
+    "unmock-jest-runner": "0.0.0",
```

Comment:
Should the node package ship with `unmock-jest-runner`?

Reply:
Based on your previous comment... probably not 😅 I'll adjust!